build.gradle

```diff
@@ -96,10 +96,8 @@ dependencies {
     compile 'com.opencsv:opencsv:3.4'
     compile 'com.google.guava:guava:18.0'
     compile 'com.github.samtools:htsjdk:'+ htsjdkVersion
-    // Using the shaded version to avoid conflicts between its protobuf dependency
-    // and that of Hadoop/Spark (either the one we reference explicitly, or the one
-    // provided by dataproc).
-    compile 'com.google.cloud:gcloud-java-nio:0.2.8:shaded'
+    // Don't use the shaded version, since it includes un-relocated netty, which causes conflicts with Spark
+    compile 'com.google.cloud:gcloud-java-nio:0.2.8'
```

Comment:
I can confirm that ExampleNioCountReads works with the old code on a v1 Dataproc cluster. Switching to this PR's code it works neither on a v1 nor a v1.1 Dataproc cluster.